content/garden/italian-penicillin/index.md

```
@@ -0,0 +1,33 @@
+
+Blend them to a smooth soup.
+
+Then throw in a parmesan rind, and cook whatever small pasta you have until al dente.
+
+Pastina are harder to find at supermarkets but I bought them online cos I love the little flowers.
+
+It works with orzo as well.
+
+Then you just have a super satisfying, filling soup. Throw in some chicken, serve it with bread.
+
+It's great!
```

markdownlint-cli2 (0.21.0)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)